x-pack/plugins/apm/public/components/routing/service_detail/index.tsx

```diff
@@ -340,16 +340,23 @@ export const serviceDetail = {
           }),
         }),
       },
-      '/services/{serviceName}/alerts': page({
-        tab: 'alerts',
-        title: i18n.translate('xpack.apm.views.alerts.title', {
-          defaultMessage: 'Alerts',
+      '/services/{serviceName}/alerts': {
+        ...page({
+          tab: 'alerts',
+          title: i18n.translate('xpack.apm.views.alerts.title', {
+            defaultMessage: 'Alerts',
+          }),
+          element: <AlertsOverview />,
+          searchBarOptions: {
+            hidden: true,
+          },
         }),
-        element: <AlertsOverview />,
-        searchBarOptions: {
-          hidden: true,
-        },
-      }),
+        params: t.partial({
+          query: t.partial({
+            alertStatus: t.string,
```

Comment:
It might make sense to have a union of values (`t.literal`) here for `ALERT_STATUS_ACTIVE`, `ALERT_STATUS_RECOVERED` etc